frontend/src/lib/components/PropertyKeyInfo.js

```diff
@@ -163,10 +163,15 @@ export const keyMapping = {
             description: 'Direct link to the exception in Sentry',
             examples: ['https://sentry.io/...'],
         },
-        $device: {
-            label: 'Device',
-            description: 'The mobile device that was used.',
-            examples: ['iPad', 'iPhone', 'Android'],
```

Comment:
I don't think we should remove this. Also, IMO we should support `$initial_device` too, as an analogy (https://github.com/PostHog/plugin-server/pull/228/#discussion_r586238960).

Reply:
It was duplicated

Reply:
![image](https://user-images.githubusercontent.com/53387/109786790-d4b54280-7c0d-11eb-94f2-2335a8bfc566.png)

Reply:
Alright, thanks